src/Microsoft.AspNetCore.OData/Microsoft.AspNetCore.OData.csproj

```diff
@@ -80,6 +80,12 @@
       <DependentUpon>SRResources.resx</DependentUpon>
     </Compile>
   </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1'">
+    <PackageReference Include="Microsoft.Netcore.app">
```

Comment:
Can we add this `PackageReference` where we have a similar condition (Line 18):
![image](https://github.com/OData/WebApi/assets/28291332/d816bd6b-34f4-4a6b-97cc-f4077e1d17db)